### DISCUSSION TITLE
Sunsetting discourse.prefect.io

### DISCUSSION THREAD

**Discussion by aaazzam** [PrefectHQ/prefect#16467] [General]
We’re retiring our Discourse forum and moving all Q&A to GitHub Discussions. This will:
- Streamline feedback and conversations in one place
- Ensure discussions stay aligned with Prefect’s latest versions
- Make it easier for our team to respond and engage

Thanks for understanding—see you here in the GitHub Discussions!

**Comment by marius-hab** [comment]
Hi, I would like to read this [discussion](https://discourse.prefect.io/t/how-to-deal-with-the-long-startup-time-latency-on-flow-runs-deployed-to-ecs-fargate/410). When looking for it on Github Discussions, I can not find it.
Have you archived this thread somewhere?

**Reply by zzstoatzz** [reply]
hi @marius-hab - I don't believe all discourse posts were archived. Would you be willing to open a new discussion here about ECS startup latency so we can add content there and reference it in the future?

**Comment by sgbaird** [comment]
@aaazzam, I often hesitate between going to slack vs. gh discussion (originally the forum). In particular, I might want to use Marvin and potentially get quicker responses on slack, but it becomes harder to find later (less searchable) and is non-provenant, due to Slack's 90-day "hiding" policy for free-tier slack accounts (makes sense, there's 17k people on the slack). So, if I try to link to one of those messages, I won't be able to view it after it becomes hidden at 90 days. On the other hand, discussions are more provenant and have other bells and whistles associated with GitHub, but might be less amenable to quick, short-form interactions.

There is a "save thread to discourse" slack app integration, which is now non-functional. Would it be possible to migrate this to open a pre-filled discussion? This would help me avoid needing to copy each individual message over, as well as having to deal with the fact that formatting gets mostly lost when manually copy-pasting from slack.

([example thread](https://prefect-community.slack.com/archives/C04DZJC94DC/p1755876187878969), posted on 2025-08-22)

<img width="1200" height="831" alt="image" src="https://github.com/user-attachments/assets/448e5b7b-e53f-4b63-bb6c-e6ba3108b1a0" />

<img width="477" height="76" alt="image" src="https://github.com/user-attachments/assets/4aea3d87-f501-45d2-9b9c-6bea123e3b56" />

**Reply by zzstoatzz** [reply]
hi @sgbaird - this is something we're thinking about recently. we'll get that old "Save to discourse" button removed, and Marvin currently has the ability to create discussions from threads (just ask it!) but we'll add a button for that as well!

totally agreed on slack being not ideal because information loss over time

**Reply by zzstoatzz** [reply]
the old "Save to Discourse" button is now removed
